src/test/java/jraft/impl/RaftServerContextImplTest.java

```diff
@@ -1,18 +1,34 @@
 package jraft.impl;
 
 import jraft.RaftServerContext;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.*;
```

Comment:
this is not a good practice. if you know what you are going to use, just refer it directly rather than involve unnecessary stuff.